### PR TITLE
making the view of transfer mobile friendly

### PR DESCRIPTION
**Pull request by felixbbertrand**
# Description

  Making the transfer view  mobile friendly.

# Test Plan

  No test has been added, since its basically a css change
  
  
  Before changes: 
image

  
  After changes :
image

  paired with @thr3m

### CODE REVIEW

**Comment by felixbbertrand** [comment]
#2646

**Comment by KuraFire** [comment]
Awesome work, both of you! From the screenshot it looks like some of the button state interactions have already been improved, too. 🥳  

Can you provide a screen capture of the interactions we discussed on a call yesterday? Or verify for me that the Transfer Grain button remains disabled until all three of From, To, and Amount have a valid value?

To recap, my requested UX improvements (since you're adjusting the Transfer Grain screen):

1. Add a visual 'arrow' pointing down (could be just a small CSS arrow using an `:after` pseudo-element ([example for this technique](http://html-tuts.com/css-arrows-up-down-left-right-triangle/)) to the Amount field. In other words, a triangle pointing down below the Amount gray area would suffice :)
2. Have the "Save Ledger" button in `:disabled` state until a Grain transfer has been submitted, at which point there is also…
3. A message above the "Save Ledger" button in a bold white color, reading: "Changes not yet saved to ledger!"

**Review by KuraFire — CHANGES_REQUESTED** [review]
See my previous comment for requested changes. Beyond those, looks great! 😎 👍🏼

**Comment by blueridger** [comment]
@KuraFire i had suggested they separate your point 2 and 3 into a separate PR, since those improvements aren't mobile-specific and really represent a distinct feature increment from what this PR accomplishes.

**Comment by felixbbertrand** [comment]
@KuraFire yeah @blueridger told us that points 2-3 should be done into another PR which makes sense.

**Comment by felixbbertrand** [comment]
pr

File: src/ui/components/Transfer.js [hunk 1]
```diff
@@ -57,6 +79,16 @@ export const Transfer = ({
...
+  const isXSmall = useMediaQuery((theme) => theme.breakpoints.down("xs"));
+
+  const disabled = (amount) => {
+    return (
+      !Number(amount) ||
+      !(sender && receiver) ||
+      gt(fromFloatString(amount), sender.balance) ||
+      lt(fromFloatString(amount), fromFloatString("0"))
+    );
+  };
```
Comment (hammadj) [inline src/ui/components/Transfer.js:91]: Why does this need to be a function if it's only used once? Can just do:

```suggestion
  const isDisabled = (
      !Number(amount) ||
      !(sender && receiver) ||
      gt(fromFloatString(amount), sender.balance) ||
      lt(fromFloatString(amount), fromFloatString("0"))
    );
```

**Review by hammadj — APPROVED** [review]
Looks good, minor nit

File: src/ui/components/Transfer.js (hunk 1 above)
Comment (felixbbertrand) [reply src/ui/components/Transfer.js]: Unsure, is there a benefit between making it a function rather than a const?

**Review by felixbbertrand — COMMENTED** [review]
@KuraFire  I'm stuck with requested changes even tho @hammadj approved these changes

**Pull request merged**